n2ws/manifest.json

```diff
@@ -24,9 +24,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-Graphicalversion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
-	  "N2WSBackup&Recovery-Graphicalversion-areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json",
-	  "N2WSBackup&Recovery-Squaresdashboard": "assets/dashboards/N2WSBackup&Recovery-Squaresdashboard.json"
+	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
+	  "N2WSBackup&Recovery-GraphicalVersion": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion.json",
+	  "N2WSBackup&Recovery-GraphicalVersion-Areas": "assets/dashboards/N2WSBackup&Recovery-Graphicalversion-areas.json"
```

Comment:
Once deployed to prod these names cannot be changed, however they are not visible inside the app and you can change the title inside the dashboard json.